index.html

```diff
@@ -66,172 +68,186 @@ <h1>Felix Odete</h1>
+<!--            <div class="card">-->
+<!--                <div class="mac-right-image"></div>&ndash;&gt;-->
+<!--                    <div class="project-section-3">-->
+<!--                        <h2>Project name goes here</h2>-->
+<!--                    </div>-->
+<!--                    <div class="grid-container-3">-->
+<!--                        <div class="html-css-rectangle">-->
+<!--                            <p>HTML/CSS</p>-->
+<!--                        </div>-->
+<!--                        <div class="ruby-rails-rectangle">-->
+<!--                            <p>Ruby on Rails</p>-->
+<!--                        </div>-->
+<!--                        <div class="javascript-rectangle">-->
+<!--                            <p class="js-text">Javascript</p>-->
+<!--                        </div>-->
+<!--                        <button type="button" class="see-project">See this project</button>-->
+<!--                        <div class="arrow"></div>-->
+<!--                    </div>-->
+<!--            </div>-->
+<!--            <div class="card">-->
+<!--                <div class="mac-left-image"></div>-->
+<!--                <div class="project-section-4">-->
+<!--                    <h2>Project name goes here</h2>-->
+<!--                </div>-->
+<!--                <div class="grid-container-4">-->
+<!--                    <div class="html-css-rectangle">-->
+<!--                        <p>HTML/CSS</p>-->
+<!--                    </div>-->
+<!--                    <div class="ruby-rails-rectangle">-->
+<!--                        <p>Ruby on Rails</p>-->
+<!--                    </div>-->
+<!--                    <div class="javascript-rectangle">-->
+<!--                        <p>Javascript</p>-->
+<!--                    </div>-->
+<!--                    <button type="button" class="see-project">See this project</button>-->
+<!--                    <div class="arrow"></div>-->
+<!--                </div>-->
+<!--            </div>-->
+<!--            <div class="card">-->
+<!--                <div class="mac-pink"></div>-->
+<!--                <div class="project-section-5">-->
+<!--                    <h2>Project name goes here</h2>-->
+<!--                </div>-->
+<!--                <div class="grid-container-5">-->
+<!--                    <div class="html-css-rectangle">-->
+<!--                        <p>HTML/CSS</p>-->
+<!--                    </div>-->
+<!--                    <div class="ruby-rails-rectangle">-->
+<!--                        <p>Ruby on Rails</p>-->
+<!--                    </div>-->
+<!--                    <div class="javascript-rectangle">-->
+<!--                        <p>Javascript</p>-->
+<!--                    </div>-->
+<!--                    <div class="see-project">-->
+<!--                        <h3>See this project</h3>-->
+<!--                        <img alt="" src="img/union.svg">-->
+<!--                    </div>-->
+<!--                </div>-->
+<!--            </div>-->
```

Comment:
- [ ] Kindly remove commented code, commented or unused code reduce readability and maintainability of your program.

Reply:
Thanks! Removed the commented code!